tests/end_to_end/test_oidc.py

```diff
@@ -8,9 +8,10 @@
 
 import requests
 import subprocess
-from lxml import html
 from urlparse import urlparse
 
+import lxml.html as html
+
```

Comment:
Is there a new python style guideline I'm missing? The `from <module> import <submodule>` seems much less stilted than `import <module>.<submodule> as <submodule>`, with exactly the same end result.